src/components/override-sdk/infra/ActionButtons/ActionButtons.tsx

```diff
@@ -41,29 +43,30 @@ export default function ActionButtons(props) {
               key={sButton.actionID}
               attributes={{ type: 'button' }}
             >
-              {localizedVal(sButton.name, localeCategory)}
+              {!isUnAuth ? localizedVal(sButton.name, localeCategory) : t('CLOSE_CLAIM')}
```

Comment:
isUnAuth ? t('CLOSE_CLAIM') : localizedVal(sButton.name, localeCategory)

Dont negate the condition unnecessarily, it becomes complex to read

Reply:
Done.